theme/src/components/do-dont.js

```diff
@@ -35,13 +35,13 @@ export function Do({src, children}) {
           Do
         </Text>
       </Flex>
-      <img src={src} width="100%" />
+      <img src={src} alt={alt || ''} width="100%" />
```

Comment:
Can we default `alt` to `''` with defaultProps so we don't have to write `|| ''` whenever we use `alt`? Something like this:

```jsx
function Do({src, alt, children}) {
   ...
   <img src={src} alt={alt} width="100%" />
   ...
}

Do.defaultProps = {
  alt: ''
}
```

Reply:
Definitely! I was also looking at refactoring into a DoDontBase. Something like 

```
export function Do(props) {
  return <DoDontBase {...props} text="Do" icon={Check} iconBg="green.5" />
}

export function Dont(props) {
  return <DoDontBase {...props} text="Don't" icon={X} iconBg="red.5" />
}
```

What do you think about that?

Reply:
DoDontBase would be a non-exported function.

Reply:
Yeah, makes sense to me. Go for it. I was also thinking maybe `DoDontContainer` would be a better name for the current `DoDont` component. What do you think?

Reply:
Yeah, I think I like that better. Want me to go ahead and update? Anything I need to do to handle the breaking change? Or is updating the theme and the same in this repo sufficient?

Reply:
Yeah, go for it. I think since we're still in major version zero (`0.y.z`), we can publish this change in a `0.15.0` release. Semver says:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Plus, I don't think anyone is using the Do/Don't components yet.